lib/http.js

```diff
@@ -116,6 +117,36 @@ function get(url, options, cb) {
   return req;
 }
 
+/**
+ * Load WebSocket class from undici
+ * @returns {void}
+ */
+function loadWebSocket() {
+  if (!WebSocket) {
+    ({ WebSocket } = require('internal/deps/undici/undici'));
+  }
+}
+
+/**
+ * Load MessageEvent class from undici
+ * @returns {void}
+ */
+function loadMessageEvent() {
+  if (!MessageEvent) {
+    ({ MessageEvent } = require('internal/deps/undici/undici'));
+  }
+}
+
+/**
+ * Load CloseEvent class from undici
+ * @returns {void}
+ */
+function loadCloseEvent() {
+  if (!CloseEvent) {
+    ({ CloseEvent } = require('internal/deps/undici/undici'));
+  }
+}
+
```

Comment:
IMO, instead of `loadXXX`, it should be `lazyXXX`, and return `XXX` for use later.

Like,
```js
function lazyCloseEvent() {
  if (!CloseEvent) {
    ({ CloseEvent, MessageEvent, WebSocket } = require('internal/deps/undici/undici'));
  }
  return CloseEvent;
}
```

And then when getting it
```js
  get: () => lazyCloseEvent();
```

Reply:
I agree with lazy sounding right, too, and I debated whether to call it lazyXXX or loadXXX but I don't have a strong opinion unless this is a convention.